### ISSUE TITLE
Basic Queries regarding MobileBERT predictions using TFLite Model Maker

### ISSUE THREAD

**Issue by katreparitosh** [tensorflow/tensorflow#43112]
Hello,

I trained the pre-trained MobileBERT using TFLite Model Maker for a binary text-classification task. The training and evaluation went fine.

I have two doubts:

1) How can the trained model be used to predict the label for a **sample sentence** which is passed as an argument to the model? When we tried the many possible methods to predict the label for a sample sentence, we got errors regarding "tensors required". Rather, how can we pass a **list of strings** to the model and **get the labels**? What is the appropriate method?

2) Also, what are the correct methods to **re-load the saved model** into a new Colab session to perform predictions? 

I would be grateful for any guidance received. Thanks.

**Comment by amahendrakar** [comment]
> When we tried the many possible methods to predict the label for a sample sentence, we got errors regarding "tensors required". Rather, how can we pass a **list of strings** to the model and **get the labels**? What is the appropriate method?

@katreparitosh,
In order to expedite the trouble-shooting process, could you please provide the complete code to reproduce the issue reported here and the dataset you are using.

Alternatively, you can also share the Colab notebook you are running. Thanks!

**Comment by katreparitosh** [comment]
Hello, I will re-frame my query because now I have a better understanding of what I need. 

I basically want to load the saved model into a script and run inference on sample sentences. Therefore, I followed the code provided in the [documentation](https://www.tensorflow.org/lite/guide/inference#load_and_run_a_model_in_python). 

The line - "input_data = np.array(np.random.random_sample(input_shape), dtype=np.int32)" feteches a random sample and later gives prediction. 

My query is **how can I write a sample sentence** and **pass** it for prediction. Example:  "What a sunny day",etc. 

Kindly share the correct procedure to do so where I can just change the sentence in the code cell and run sample tests?

**Comment by amahendrakar** [comment]
> My query is **how can I write a sample sentence** and **pass** it for prediction. Example: "What a sunny day", etc. I was unable to find references where TFLite MobileBERT is used for Text Classification.

@katreparitosh,
Please use the same pre-processing steps you have used during the training, on the sentence you want to pass and then pass it as the input to the model. Please take a look at [this](https://www.tensorflow.org/tutorials/keras/text_classification#prepare_the_dataset_for_training) text classification tutorial which might be useful in this case.

Also, this question is better asked on [StackOverflow](https://stackoverflow.com/questions/tagged/tensorflow) since it is not a TensorFlow bug or feature request. There is also a larger community that reads questions there. Thanks!
